packages/camera/camera_android/lib/src/android_camera.dart

```diff
@@ -92,19 +92,21 @@ class AndroidCamera extends CameraPlatform {
   }
 
   @override
```

Comment:
Shouldn't we deprecate `createCamera`?
At this point, there is no need to override `createCamera`, because `platform interface` just calls `createCameraWithSettings`.
dummy override producing analyzer warnings:
![image](https://user-images.githubusercontent.com/8981380/234512164-f235965d-78bd-4415-9abd-071bcd14ea71.png)